routing/pathfind.go

```diff
@@ -705,11 +714,19 @@ func findPath(g *graphParams, r *RestrictParams, cfg *PathFindingConfig,
 			return
 		}
 
+		// If this is one of our own channels and 
+		// LocalOpportunityCost is true, then we account for the
+		// fee on this channel.
+		var opportunityCostFee lnwire.MilliSatoshi
+		if cfg.LocalOpportunityCost && fromVertex == source {
```

Comment:
i was looking at doing something like that, but couple issues:
- i dont think we want to edit `fee` because that is used to calculate `totalFee` on line 700 which might affect the actual amount sent, which is not what we want (because we don't actually pay this new `opportunityCostFee`)
- we don't want to add the timeLockDelta of this edge to our total timelock calculation for a similar reason

Reply:
gotcha